compiler/resolution/fcf-support.cpp

```diff
@@ -316,6 +320,11 @@ buildWrapperSuperTypeAtProgram(const std::vector<FcfFormalInfo>& formals,
   v->thisMethod = attachSuperThis(v->type, formals, retTag,
                                   retType,
                                   throws);
+  std::ignore = attachSuperWriteMethod(v->type, "writeThis");
+  if (fUseIOFormatters) {
+    std::ignore = attachSuperWriteMethod(v->type, "writeThis");
```

Comment:
It looks like you're calling this function in the same way twice.  Should this second call say "encodeTo"?

Reply:
Yeah, should call `encodeTo`. Thanks!